ui/app/src/services/content.ts

```diff
@@ -14,7 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { del, errorSelectorApi1, get, getText, post, postJSON } from '../utils/ajax';
+import { del, errorSelectorApi1, get, getGlobalHeaders, getText, post, postJSON } from '../utils/ajax';
```

Comment:
Is this still used? Check console warnings (compile/build).

Reply:
this one is, used in createFileUpload (uses Uppy)